.papr.yml

```diff
@@ -169,3 +169,22 @@ tests:
   - git clone https://github.com/projectatomic/atomic-host-tests
   - make vmoverlay HOSTS=testnode
   - cd atomic-host-tests && ./.test_director
+
+---
+
+branches:
+  - master
+  - auto
+  - try
+
+cluster:
+  hosts:
+    - name: composetest-f27
+      distro: fedora/27/atomic
```

Comment:
This should be CentOS, right?

Reply:
I started out doing that but...it's a lot more of a pain to set up as we'd need to inject the sig-atomic repos into the mock config etc.

Reply:
And I believe 93.2% of what we're testing here is running inside `mock --old-chroot` - if there are any issues past that we should IMO do a separate PR.